kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1143,6 +1143,16 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(false)
 
+  val ENGINE_USER_ISOLATED_SPARK_SESSION: ConfigEntry[Boolean] =
+    buildConf("kyuubi.engine.user.isolated.spark.session")
+      .doc("When set to true, if the engine is running in a group or server share level, " +
```

Comment:
false, anything different when false? not isolated?

Reply:
yes if false, all users within one group/server are not isolated